.github/workflows/bindings-ts.yml

```diff
@@ -0,0 +1,40 @@
+name: bindings typescript
+
+on: [push]
```

Comment:
I think you need the pull request trigger specified so that the build will run on PRs.
```suggestion
on:
  push:
    branches: [main, release/**]
  pull_request:
```

Reply:
Thanks! [There it is](https://github.com/stellar/soroban-tools/actions/runs/6027754051/job/16353628501?pr=905)